docs/source/conf.py

```diff
@@ -147,14 +147,15 @@
 project = 'PyTorch'
 copyright = '2019, Torch Contributors'
 author = 'Torch Contributors'
+torch_version = str(torch.__version__)
```

Comment:
To fix CI failures related to:

```
make: *** [html-stable] Error 2
++ code=2
++ '[' 2 -ne 0 ']'
++ set +x
=========================
WARNING: The config value `version' has type `TorchVersion', defaults to `str'.
WARNING: The config value `release' has type `TorchVersion', defaults to `str'.
=========================
Docs build failed. If the failure is not clear, scan back in the log
for any WARNINGS or for the line build finished with problems
(tried to echo the WARNINGS above the ==== line)
=========================
++ return 2
```

See: https://github.com/pytorch/pytorch/runs/3052078756?check_suite_focus=true